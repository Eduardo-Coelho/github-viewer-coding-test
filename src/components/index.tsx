export * from './commitDetails/CommitDetails';
export * from './commitTable/CommitTable';
export * from './repoSearch/RepoSearch';
export * from './loader/Loader';
export * from './modal/Modal';
