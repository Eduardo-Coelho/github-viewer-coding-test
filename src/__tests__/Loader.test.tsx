import React from 'react';
import { render } from '@testing-library/react';
import { Loader } from '../components';

test('Loader matches snapshot', () => {
  const { asFragment } = render(<Loader />);
  expect(asFragment()).toMatchSnapshot();
});
