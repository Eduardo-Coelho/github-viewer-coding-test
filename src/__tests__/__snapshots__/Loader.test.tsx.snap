// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="loading-container"
  >
    <img
      alt="loading..."
      src="XVo6.gif"
    />
  </div>
</DocumentFragment>
`;
